src/encoded/schemas/patient.json

```diff
@@ -216,6 +220,9 @@
         "diagnosis.age_unit": {
             "title": "Diagnosis age units"
         },
+        "duration_of_follow_up_range": {
```

Comment:
"duration_of_follow_up_range" is used in "columns" which is different then facet "diagnosis.follow_up_duration_range". One of them is wrong?

Reply:
I forgot to add a update field in columns. I will add them and repush. Thanks.

Reply:
Looks good